src/components/Table/styles.module.css

```diff
@@ -0,0 +1,24 @@
+.table-wrapper {
+    display: grid;
+    justify-content: center;
+    grid-template-columns: repeat(var(--columns), 1fr);
+    grid-gap: .5rem;
+    font-family: Helvetica;
+    text-align: center;
+    border: .25rem solid white;
+    background-color: white;
+}
+
+.table-header-item,
+.table-body-item {
+    padding: .75rem;
+}
+
+.table-header-item {
+    background-color: var(--tp-primary-blue);
+    font-weight: bold;
+}
+
+.table-body-item {
+    background-color: black;
```

Comment:
Can we omit the background and go with transparent for the body items? I'd like whatever background is on the page to come through.

Reply:
By omitting the background, I can come up with a better solution for the remainder cell space.

A triangle after the table item when there is a leftover space in the table LGTM.
![Table screenshot](https://user-images.githubusercontent.com/2235293/160476147-fb0a0ddd-bd41-4de5-a611-45d97bff325b.png)